packages/office-ui-fabric-react/src/components/Check/Check.classNames.ts

```diff
@@ -0,0 +1,4 @@
+import { classNamesFunction } from '../../Utilities';
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+
+export const getClassNames = classNamesFunction<ICheckStyleProps, ICheckStyles>();
```

Comment:
Convention for components so far is to have this in their base file... is there a reason to break this out into its own file?

Reply:
I was sharing this and realized I don't need to. I'll move it back